src/v2/components/Validators/Detail/index.jsx

```diff
@@ -0,0 +1,198 @@
+// @flow
+import {Container, useTheme} from '@material-ui/core';
+import {observer} from 'mobx-react-lite';
+import useMediaQuery from '@material-ui/core/useMediaQuery/useMediaQuery';
+import React from 'react';
+import {map, find, compose, mergeWith} from 'lodash/fp';
+import {Match} from 'react-router-dom';
+import {
+  ComposableMap,
+  Geographies,
+  Geography,
+  Marker,
+  Markers,
+  ZoomableGroup,
+} from 'react-simple-maps';
+import SectionHeader from 'v2/components/UI/SectionHeader';
+import NodesStore from 'v2/stores/nodes';
+import {mapStyle, markerStyle} from 'v2/theme';
+import MapTooltip from 'v2/components/UI/MapTooltip';
+import Bx2HelpLink from 'v2/Bx2HelpLink';
+
+import useStyles from './styles';
+
+const mapStyles = {
+  default: mapStyle,
+  hover: mapStyle,
+  pressed: mapStyle,
+};
+
+const ValidatorsDetail = ({match}: {match: Match}) => {
+  const {
+    cluster: {voting, cluster},
+  } = NodesStore;
+  const classes = useStyles();
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+  const {params} = match;
+  const currentCluster = find({pubkey: params.id})(cluster);
+  if (!currentCluster) {
+    return <div>Loading...</div>;
+  }
+  const node = compose(
+    mergeWith(currentCluster, {
+      coordinates: [currentCluster.lng, currentCluster.lat],
+    }),
+    find({nodePubkey: params.id}),
+  )(voting);
+
+  const renderMarker = () => (
+    <Marker style={markerStyle} marker={node}>
+      <MapTooltip
+        classes={{tooltip: classes.tooltip}}
+        title={() => (
+          <>
+            <div className={classes.tooltipTitle}>NODE: {node.nodePubkey}</div>
+            <div className={classes.tooltipDesc}>Gossip: {node.gossip}</div>
+          </>
+        )}
+      >
+        <circle
+          cx={0}
+          cy={0}
+          r={5}
+          style={{
+            stroke: '#00FFAD',
+          }}
+        />
+      </MapTooltip>
+    </Marker>
+  );
+
+  const specs = [
+    {
+      label: 'Website',
+      hint: '',
+      value: 'solana.com',
+    },
+    {
+      label: 'voting power',
+      hint: '',
+      value: '286,431',
+    },
+    {
+      label: 'Address',
+      hint: '',
+      value: 'cosmos1sjllsnramtg3ewxqwwrwjxfgc4n4ef9u0tvx7u',
+    },
+    {
+      label: 'Missed blocks',
+      hint: '',
+      value: '43',
+    },
+    {
+      label: 'keybase',
+      hint: '',
+      value: 'A21D897660B856AC',
+    },
+    {
+      label: 'commission',
+      hint: '',
+      value: `${node.commission}%`,
+    },
+    {
+      label: 'details',
+      hint: '',
+      value:
+        'Operated by Chris Remus (Twitter @cjremus) / Validating since the Validator Working Group formed in October 2017',
+    },
+    {
+      label: 'Amount of delegated sol',
+      hint: '',
+      value: '5.47%',
```

Comment:
I think this is `.stake` compared to "sum of all active stakes" expressed as a percentage

Reply:
@sunnygleason ideally should be provided by api.

Reply:
put TODO for now